impeller/renderer/render_pass.h

```diff
@@ -115,11 +121,9 @@ class RenderPass : public ResourceBinder {
   ///
   /// @param[in]  vertex_buffer  The buffer view to use for sourcing vertices.
   ///
-  /// @param[in]  vertex_count   The number of vertices to draw.
-  ///
   /// @return     Returns false if the given buffer view is invalid.
   ///
-  bool SetVertexBuffer(BufferView vertex_buffer, size_t vertex_count);
+  bool SetVertexBuffer(BufferView vertex_buffer);
```

Comment:
This patch removes `vertex_count` from `SetVertexBuffer`. It gets set via `SetElementCount` now. (This value becomes the index count when an index buffer is present)